### PR TITLE
Conversie van een Bedrag naar centen

### PR DESCRIPTION
**Pull request by svroonland**

### CODE REVIEW

File: src/main/scala/org/scalarules/finance/nl/Bedrag.scala [hunk 1]
```diff
@@ -41,6 +41,9 @@ case class Bedrag private[finance] (waarde: BigDecimal) {
   def afgerondOp(aantalDecimalen: Integer, afrondingsWijze: RoundingMode): Bedrag =
     Bedrag(waarde.setScale(aantalDecimalen, afrondingsWijze))
 
+  /** Returnt het aantal centen dat het bedrag vormt */
```
Comment (svroonland) [inline src/main/scala/org/scalarules/finance/nl/Bedrag.scala]: Fair enough. After talking to Jan, we decided to bail on this PR :)

We're going  to implement this in our own code adapted to our specific needs. If a nice generic solution follows from this, there may be another PR.

This PR can be closed.

Comment (NRBPerdijk) [inline src/main/scala/org/scalarules/finance/nl/Bedrag.scala]: Sounds like a plan! :)
